_layouts/default.html

```diff
@@ -26,7 +26,7 @@
 
     <link rel="stylesheet" href="{{ site.cdnurl }}{{ site.baseurl }}/css/pygments-github.css">
     <link rel="stylesheet" href="{{ site.baseurl }}/css/vendors.css">
-    <link rel="stylesheet" href="{{ site.baseurl }}/synapse/dist/scoped-synapse.css">
+    <link rel="stylesheet" href="{{ site.baseurl }}/css/scoped-synapse.css?v=2">
```

Comment:
So if there is any change to synapse css, it looks like we have to copy in the rebuilt css, eh? Should something be added to the Readme about synapse css?

Reply:
Nice catch, I'll update.

Reply:
Updated!